From 58beeb4837da3c726a886e8ea20eca854084fdb0 Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Fri, 8 Mar 2019 15:36:21 +0100
Subject: Remove unused dependencies

These seem to be leftovers from plasma-pa

Differential Revision: https://phabricator.kde.org/D19607
---
 CMakeLists.txt | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3aabeb6..5ea8caf 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -67,11 +67,6 @@ configure_package_config_file(
   INSTALL_DESTINATION  ${CMAKECONFIG_INSTALL_DIR}
 )
 
-find_package(KF5 ${KF5_MIN_VERSION} REQUIRED COMPONENTS
-    CoreAddons
-    DocTools
-    I18n
-)
 find_package(PulseAudio 5.0.0 REQUIRED)
 find_package(GLIB2 REQUIRED)
 
-- 
cgit v1.1
